Fakebook.Profile/Fakebook.Profile.UnitTests/APITests/ProfileIntegrationTests.cs

```diff
@@ -1,7 +1,115 @@
-﻿namespace Fakebook.Profile.UnitTests.APITests
+﻿using System;
+using System.Linq;
+using System.Collections.Generic;
+using Xunit;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Fakebook.Profile.Domain;
+using Fakebook.Profile.DataAccess;
+
+namespace Fakebook.Profile.UnitTests.APITests
 {
     public class ProfileIntegrationTests
     {
+        [Fact]
+        public async Task ProfileRepoAddProfile()
+        {
+            using var contextFactory = new ContextFactory();
+            using var context = contextFactory.CreateContext();
+
+            DomainProfile test = new DomainProfile("tdunbar@google.com", "Trevor", "Dunbar");
+            test.BirthDate = new DateTime(1994, 6, 30, 0, 0, 0, 0, DateTimeKind.Unspecified);
+
+            var repo = new ProfileRepository(context);
+            await repo.CreateProfileAsync(test);
+
+            var compare = context.EntityProfiles.Local.Single(x => x.Id == 3); //ID is 3 because there's already 2 users in seed data
```

Comment:
test is brittle - we're guaranteed to be modifying that seed data. instead, write a line or two in the arrange step to calculate what the ID should be from the context object. likewise the other tests that do the same thing. maybe have a helper method for it.

Reply:
Removed magic numbers for Ids and added int variable that should grab the ID of whatever was added to the DB last.